src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -34,17 +34,18 @@ public class Sender : EndpointConfigurationBuilder
         {
             public Sender()
             {
-                var basePath = AppDomain.CurrentDomain.BaseDirectory;
-
-                File.WriteAllLines(Path.Combine(basePath, "DistributingACommand.Receiver.txt"), new[]
-                {
-                    "1:",
-                    "2:"
-                });
+                var filePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "routes.xml");
+                File.WriteAllText(filePath, @"<endpoints>
+    <endpoint name=""DistributingACommand.Receiver"">
+        <instance scaleOutDiscriminator=""1""/>
```

Comment:
>I like ~~scaleOut. This's used only for that and it's more expressive than just ~~discriminator

Not sure I agree, instances and how to discriminate them is by definition all about scaling out so I don't see what `scaleOut` adds